vhdbuilder/packer/install-dependencies.sh

```diff
@@ -133,6 +133,11 @@ if [[ ${CONTAINER_RUNTIME:-""} == "containerd" ]]; then
   for version in $containerd_versions; do
     containerd_version="$(echo "$version" | cut -d- -f1)"
     containerd_patch_version="$(echo "$version" | cut -d- -f2)"
+    # containerd 1.4 not available in ubuntu 22.04
+    if [[ ${UBUNTU_RELEASE} == "22.04" && ${containerd_version} == "1.4.13" ]]; then
```

Comment:
let's just stop precaching this for all cases? but this is fine for now

Reply:
that makes sense, ill do it in a separate PR